test/e2e/addon_update.go

```diff
@@ -283,6 +284,7 @@ var _ = framework.KubeDescribe("Addon update", func() {
 		sshExecAndVerify(sshClient, fmt.Sprintf("sudo cp %s/%s %s/%s", temporaryRemotePath, svcv2, destinationDir, svcv2))
 		sshExecAndVerify(sshClient, fmt.Sprintf("sudo rm %s/%s", destinationDir, rcv1))
 		sshExecAndVerify(sshClient, fmt.Sprintf("sudo rm %s/%s", destinationDir, svcv1))
+		sshExecAndVerify(sshClient, fmt.Sprintf("sudo pkill -9 -o -f kube-addon"))
```

Comment:
oops...I misread your change. I thought you were changing the period to 1 minute.
Actually, would changing this to 1min (instead of 10min) enough? If we kill the pod every time, the test is not testing the regular behavior of the addon manager.